CHANGELOG.md

```diff
@@ -20,6 +19,9 @@ return `Option<Strings>` instead of opaque `Robj`.
 
 ### Deprecated
 
+- Removed `from_sexp_ref` from public API. This is a completely unsound function, and uses of it must not
```

Comment:
Can you rephrase this please? I think something is not right in this sentence.

Reply:
What about: 

```
- Removed `from_sexp_ref` from the public API. `from_sexp_ref()` is an unsafe method and is intended for use only within `extendr-api`.